### DISCUSSION TITLE
Change Chinese to English?

### DISCUSSION THREAD

**Discussion by manhtrv** [klinecharts/KLineChart#146] [General]
Some labels are Chinese, it too complex to understand.

**Comment by liihuu** [comment]
My english is too poor. 😂

**Reply by AbdaalHassan** [reply]
how we change the default language as english in klinepro

**Comment by manhtrv** [comment]
So sad for anyone like me who don't know Chinese :(

**Comment by dooee** [comment]
How about supporting multi-language? Could this be quickly done with the current structure?

**Reply by liihuu** [reply]
Chart support multiple languages.

**Reply by n-bhasin** [reply]
how can change the language to english? it is displaying the text in chinese
